src/paymentAPI.yaml

```
@@ -321,6 +327,7 @@ paths:
         - payments
       summary: Initiate new payments submission
       description: Create new payments submission.
+      operationId: submitPaymentInstructions
```

```suggestion
      operationId: submitPaymentInstruction
```
In my opinion, it should be singular, but i'm not entirely sure.